src/PermissionManagerInterface.php

```diff
@@ -31,6 +32,25 @@ public function getPermissionList($entity_type_id, $bundle_id);
    * @return array
    *   An array of permission names and descriptions.
    */
-  public function generateCrudPermissionList($group_content_entity_type_id, $group_content_bundle_id);
+  public function generateEntityOperationPermissionList($group_content_entity_type_id, $group_content_bundle_id);
+
+  /**
+   * Returns permissions that are enabled by default for the given role.
+   *
+   * @param string $group_entity_type_id
+   *   The entity type ID of the group for which to return permissions.
+   * @param string $group_bundle_id
+   *   The bundle ID of the group for which to return permissions.
+   * @param array $group_content_bundle_ids
```

Comment:
`getDefaultPermissions()` returns all permissions that are enable by default for a given role. This is used when a new group type is created: we need to populate a default set of permissions initially, mainly to give non-members the right to subscribe, members the right to leave, and administrators the right to do everything.

For the moment this covers both the group-level permissions and the entity operation permissions. I agree that the entity operation permissions don't make sense when creating a new group type, since the group won't have any group content types yet. I don't remember why I put this in initially. I'll see if I can remove this parameter.